modules/tinymce/src/themes/silver/main/ts/ui/toolbar/button/ToolbarButtons.ts

```diff
@@ -172,6 +172,15 @@ const renderCommonToolbarButton = <T>(spec: GeneralToolbarButton<T>, specialisat
           onControlAttached(specialisation, editorOffCell),
           onControlDetached(specialisation, editorOffCell)
         ]),
+        ...(
+          spec.tooltip.map(
+            (t) => Tooltipping.config(
+              providersBackstage.tooltips.getConfig({
+                tooltipText: t
```

Comment:
I think there are probably a few places I've missed translating like here.

- [ ] sweep for missed translations